Makefile

```diff
@@ -44,7 +34,7 @@ lint: ## Verifies `golint` passes.
 .PHONY: test
 test: ## Runs the go tests.
 	@echo "+ $@"
-	@$(GO) test -v -tags "$(BUILDTAGS) cgo" $(shell $(GO) list ./... | grep -v vendor)
+	$(V)GO111MODULE=on $(GO) test -v -tags "$(BUILDTAGS) cgo" $(shell $(GO) list ./... | grep -v vendor)
```

Comment:
This can be simplified with `vendor` out of the picture:

```
	$(V)GO111MODULE=on $(GO) test -v -tags "$(BUILDTAGS) cgo" ./...
```

This applies in several places in this file, I think wherever you are doing `go list` with `grep -v vendor` you can simplify.

Reply:
22760d744523fa71ae56a070e0fc69e3c71e43f5